### ISSUE TITLE
Create-react-app-uifabric logo image not found.

### ISSUE THREAD

**Issue by ghost** [microsoft/create-react-app-uifabric#7]
When creating a new application the logo that comes from <https://raw.githubusercontent.com/Microsoft/just/master/packages/just-stack-uifabric/template/src/components/fabric.png> gives a 404 error. 

![image](https://user-images.githubusercontent.com/51803583/73494256-cd23ca00-4379-11ea-8263-b3618642993b.png)

![image](https://user-images.githubusercontent.com/51803583/73494309-e462b780-4379-11ea-8ffd-699ca7711a7c.png)

**Comment by bluebrown** [comment]
Came here to say the same, not sure which image was there but we could just do this. Looks fine.

```tsx
<img
     src="https://user-images.githubusercontent.com/785361/50458071-45b58d00-0915-11e9-90c0-ad8789c99db5.png"
     alt="logo"
     height='360'
/>
```

It is the image from the frabric-react main repo https://github.com/OfficeDev/office-ui-fabric-react

**Comment by Hotell** [comment]
fixed by #22
